src/Synth/ADnote.cpp

```diff
@@ -1776,7 +1776,10 @@ inline void ADnote::ComputeVoiceOscillatorWaveTableModulation(int nvoice)
 
     // WaveTable-specific code
     for(int k = 0; k < vce.unison_size; ++k) {
-        assert(vce.OscilSmp.isWaveTable);
+        if(!vce.OscilSmp.isWaveTable) {
+           printf("\n\n assertion isWaveTable would have failed \n");
```

Comment:
I assume you changed this because the assert was too annoying? But without the assert, won't the code give you a memory error?

Reply:
> I assume you changed this because the assert was too annoying? But without the assert, won't the code give you a memory error?

at least I didn't notice. actually i didn't look for the print output,  I guess we should prevent this path to be executed if the assertion would fail.

Reply:
oh, now i see. because of the continue in line 1781 it doesnt do anything in this case

Reply:
i changed it back, to keep this commit a single feature one.